client/adventureMap/CInGameConsole.cpp

```diff
@@ -155,7 +155,7 @@ void CInGameConsole::keyPressed (const SDL_Keycode & key)
 			}
 			break;
 		}
-	case SDLK_UP: //up arrow
+	case EShortcut::SELECT_UP:
```

Comment:
why is it SELECT_? Maybe MOVE_ or SCROLL_?

Reply:
MOVE might be better... Okay, will rename slider-related hotkeys.